### ISSUE TITLE
"Build failed with errors" message instead of webpack cause

### ISSUE THREAD

**Issue by Toilal** [vuejs-templates/webpack#1155]
When webpack build fails (npm build), it displays "Build failed with errors" without any other explanation.

Problem is that this piece of code hide actual errors.

https://github.com/vuejs-templates/webpack/blob/ecd68c4b72c0282bff1cd291ff9f91e30ce27d22/template/build/build.js#L22-L28

If is set this to `toString({color: true})`, errors are displayed with other informations.

**Comment by LinusBorg** [comment]
On what kind of errors does this happen?

I tried:

*  a syntax error wiht eslint enabled: shows eslint errors
* as syntax error with eslint disabled: shows error with highlighted code-snippet.
* a from path on an `import`: shows "module not found ..." error

Please provide more information.

**Comment by Toilal** [comment]
`import { Vue } from 'vue/types/vue'` instead of `import vue from 'vue'`

**Comment by Toilal** [comment]
Sorry, I forgot one important thing ! The error is caused by ts-loader (i'm running typescript).

**Comment by LinusBorg** [comment]
<img width="469" alt="bildschirmfoto 2017-12-11 um 16 04 55" src="https://user-images.githubusercontent.com/1444526/33837570-11a9c796-de8d-11e7-9c54-78a0abe8f7cd.png">

**Comment by LinusBorg** [comment]
> Sorry, I forgot one important thing ! The error is caused by ts-loader (i'm running typescript).

Not an unimportant detail ^^

**Comment by LinusBorg** [comment]
Please provide a repository with a reproduction, I won't build one myself.

**Comment by Toilal** [comment]
https://github.com/pragmasphere/vue-tree

It's fixed in this repository, but you revert those lines

https://github.com/pragmasphere/vue-tree/blob/2302c9b18b7298a0e83ad18f43d2f212bb487078/build/build.js#L22-L24

**Comment by LinusBorg** [comment]
Thanks. Seems ts-loader is a weird beast. In webpack 1, it even made webpack exit with code 0 on a compile error (https://github.com/TypeStrong/ts-loader/issues/108), so getting errors out of ts-loader seems to not be that easy :-P

So I played around a little and found that removing `children: false` is enough to make the error appear again.

The thing is that this alone makes the output quite verbose on a successful build. and since this template is not laid out for typescript yet, I would hesitate to make the output verbose for everyone just to accomodate those who implement typescript into that template on their own.

So my suggetion would be the following:

1. Add a comment in `build.js` to tip off people using typescript how to fix it.
2. If /when we make typescript an option for the template, with e.g. #797, we can make this option change depending on wether typescript was added or not.

Does that sound sensible to you?
